ironfish-rust/src/frost_utils/signing_package.rs

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use crate::{errors::IronfishError, transaction::unsigned::UnsignedTransaction};
+use ironfish_frost::frost::SigningPackage as FrostSigningPackage;
+
+#[derive(Clone)]
+pub struct SigningPackage {
+    pub unsigned_transaction: UnsignedTransaction,
+    pub frost_signing_package: FrostSigningPackage,
+}
+
+impl SigningPackage {
+    pub fn serialize(&self) -> Result<Vec<u8>, IronfishError> {
+        let serialized_frost_signing_package = self.frost_signing_package.serialize()?;
+        let length_bytes = (serialized_frost_signing_package.len() as u32).to_le_bytes();
```

Comment:
Nit: although unlikely, this cast can truncate the length. I'd use `TryFrom` or `TryInto` for safety:

```rust
u32::try_from(serialized_frost_signing_package.len())
```